README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
I think that having the command align with the Python CLI as much as reasonable makes sense, and was raised by someone else. I realise that it is more work, but I think it is worth it. Putting the effort aside, let's ask ourselves what the best approach would be. WDYT?